### PR TITLE
Support for axis options (Intersection Point & Reverse Axis) in PowerPoint2007 Writer

### PR DESCRIPTION
**Pull request by mindline-analytics**
- adds crosses/ crossesAt options to axis component
- adds reversed order to axis component

### CODE REVIEW

**Comment by mindline-analytics** [comment]
Hi @Progi1984, any feedback/ change requests on this PR?

File: docs/changes/1.0.0.md [hunk 1]
```diff
@@ -17,6 +17,10 @@
 - PowerPoint2007 Reader : Load images in their initial format (and not by default in PNG) - [@polidog](https://github.com/polidog) GH-553
 
 ## Features
+- Support for axis options - [@mindline](https://github.com/mindline-analytics) GH-676
+    - axis intersection point (axis crossing) through `setCrossesAt`/ `getCrossesAt`
+    - reverse axis order through `setIsReversedOrder`/ `isReversedOrder`
+    - PowerPoint2007 Writer
```
Comment (Progi1984) [inline docs/changes/1.0.0.md:23]: Could you move it to 1.1.0.md, please ?

File: docs/usage/shapes/chart.md [hunk 2]
```diff
@@ -95,6 +95,57 @@ $shape->getPlotArea()->getAxisX()->setMinBounds(0);
 $shape->getPlotArea()->getAxisX()->setMaxBounds(200);
 ```
 
+#### Crossing
+
+For Axis, you can define where it should be crossed by the perpendicular/ horizontal axis trought `setCrossesAt`.
```
Comment (Progi1984) [inline docs/usage/shapes/chart.md:100]:
```suggestion
#### Crossing


!!! warning
    Available only on the PowerPoint2007 Writer

For Axis, you can define where it should be crossed by the perpendicular/ horizontal axis trought `setCrossesAt`.
```

**Review by Progi1984 — COMMENTED** [review]
@mindline-analytics Good job. Could you rebase your PR and fix my suggestions ?

After that, I will test in my local environment and I will merge it if it's OK.

Thanks again for your contribution.

**Comment by mindline-analytics** [comment]
Hi @Progi1984, welcome back :)
I have made the requested changes. I hope Saturday's github outage caused the failing tests.

**Comment by Progi1984** [comment]
Thanks @mindline-analytics

**Pull request merged**